google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -325,6 +325,52 @@ def time_partitioning_expiration= expiration
           patch_gapi! :time_partitioning
         end
 
+        ###
+        # Whether queries over this table require a partition filter that can be
+        # used for partition elimination to be specified. See [Partitioned
+        # Tables](https://cloud.google.com/bigquery/docs/partitioned-tables).
+        #
+        # @return [Boolean, nil] `true` when a partition filter will be
+        #   required, `false` otherwise, or `nil` if the object is a reference
+        #   (see {#reference?}).
+        #
+        # @!group Attributes
+        #
+        def require_partition_filter
```

Comment:
I'm sure you considered this, but why not `require_partition_filter?` ?

Reply:
Because it is both a getter and a setter.